proto/viam/service/slam/v1/slam.proto

```diff
@@ -11,22 +11,6 @@ option java_package = "com.viam.service.slam.v1";
 
 // A SlamService declares the gRPC contract for a slam service
 service SLAMService {
-  // GetPosition returns the current estimated position of the robot with
```

Comment:
[note to self] the change to slam.proto is the only one we actually typed; the changes to the rest of the files in this PR were auto-generated based on that